src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/killaura/features/KillAuraVelocityHit.kt

```diff
@@ -0,0 +1,123 @@
+package net.ccbluex.liquidbounce.features.module.modules.combat.killaura.features
+
+import net.ccbluex.liquidbounce.config.types.ToggleableConfigurable
+import net.ccbluex.liquidbounce.event.events.PacketEvent
+import net.ccbluex.liquidbounce.event.events.TransferOrigin
+import net.ccbluex.liquidbounce.event.sequenceHandler
+import net.ccbluex.liquidbounce.event.tickHandler
+import net.ccbluex.liquidbounce.features.module.modules.combat.killaura.ModuleKillAura
+import net.ccbluex.liquidbounce.features.module.modules.misc.ModulePacketLogger
+import net.ccbluex.liquidbounce.utils.client.Chronometer
+import net.ccbluex.liquidbounce.utils.client.PacketQueueManager
+import net.ccbluex.liquidbounce.utils.client.chat
+import net.minecraft.entity.LivingEntity
+import net.minecraft.network.packet.c2s.play.ClientCommandC2SPacket
+import net.minecraft.network.packet.c2s.play.PlayerMoveC2SPacket
+import net.minecraft.network.packet.s2c.play.*
+
+object KillAuraVelocityHit : ToggleableConfigurable(ModuleKillAura, "VelocityHit", false) {
+
+    private var considerVelocityHit = false;
+    private var damageReceived = false;
+    private var onGroundTicks = 0;
+    private var isPossible = false;
```

Comment:
```suggestion
    private var considerVelocityHit = false
    private var damageReceived = false
    private var onGroundTicks = 0
    private var isPossible = false
```
semicolons :skull:
You don't need semicolons at the end of each statement in Kotlin.